### ISSUE TITLE
Binaries don't handle paths the same way

### ISSUE THREAD

**Issue by gabriel-vasile** [tinode/chat#652]
### Your environment 
@master
### Steps to reproduce
2 binaries:
/bin/tinode
/bin/init-db
```bash
$ ls .
config

$ init-db --config=config
Database exists, DB version is correct. All done.

tinode --config=config
Failed to read config file: open /bin/config: no such file or directory
```

### Expected behaviour
Both commands should work or both commands should fail with "no such file or directory".

### Actual behaviour
init-db works, tinode doesn't.

https://github.com/tinode/chat/blob/01a463ead4367d99363a05bcdef2633b64517e80/server/main.go#L276-L288
https://github.com/tinode/chat/blob/01a463ead4367d99363a05bcdef2633b64517e80/tinode-db/main.go#L176-L196

**Comment by aforge** [comment]
I made this: https://github.com/tinode/chat/pull/654
I'll let @or-else take a look.

**Comment by gabriel-vasile** [comment]
Any reasoning for why paths are relative to binary location?
It's definitely surprising for new users. Don't think I've ever seen other binaries work like this.

**Comment by or-else** [comment]
Released in 0.17.1-rc1
